tests/system_tests/test_cli.py

```diff
@@ -28,25 +27,49 @@ def phospho_files():
     return pin, fasta
 
 
-@pytest.fixture
-def pepxml_file():
-    """Get the pepxml file"""
-    pepxml = Path("data", "msfragger.pepXML")
-    return pepxml
+def count_lines(path: Path):
+    """Count the number of lines in a file.
+
+    Parameters
+    ----------
+    path : Path
+        The path to the file.
+
+    Returns
+    -------
+    int
+        The number of lines in the file.
+    """
+    with open(path, "r") as file:
+        lines = file.readlines()
+    return len(lines)
 
 
 def test_basic_cli(tmp_path, scope_files):
     """Test that basic cli works."""
-    cmd = ["mokapot", scope_files[0], "--dest_dir", tmp_path]
-    subprocess.run(cmd, check=True)
-    assert Path(tmp_path, "mokapot.psms.txt").exists()
-    assert Path(tmp_path, "mokapot.peptides.txt").exists()
+    params = [scope_files[0], "--dest_dir", tmp_path]
+    run_mokapot_cli(params)
+    assert Path(tmp_path, "targets.psms").exists()
+    assert Path(tmp_path, "targets.peptides").exists()
+
+    targets_psms_df = pd.read_csv(
+        Path(tmp_path, "targets.psms"), sep="\t", index_col=None, nrows=1
+    )
+    assert targets_psms_df.columns.values.tolist() == [
+        "PSMId",
+        "peptide",
+        "score",
+        "q-value",
+        "posterior_error_prob",
+        "proteinIds",
+    ]
+    assert targets_psms_df.iloc[0, 0] == "target_0_11040_3_-1"
+    assert targets_psms_df.iloc[0, 5] == "sp|P10809|CH60_HUMAN"
```

Comment:
Could you also add an expected length (or range of lengths..) to the assertions?

Reply:
I think its fine for now ... we can think of an upper limit as well as a smoke test (getting 10% more is probably normal, identifying 100% more is probably a bug :P)